posts/2020-02-12-using-dotty-union-types-with-akka-typed-part-II.adoc

```diff
@@ -0,0 +1,285 @@
+= Using Dotty Union types with Akka Typed - Part II
+eloots
+v1.0, 2020-02-19
+:title: Using Dotty Union types with Akka Typed - Part II
+:tags: [dotty, akka]
+
+== Introduction
+
+After posting the first article on https://www.lunatech.com/2020/02/using-dotty-union-types-with-akka-typed[Using Dotty Union types with Akka Typed], I received https://twitter.com/hseeberger/status/1227660145125199877?s=20[valuable feedback] via Twitter which made me decide to write this follow-up article.
+
+I will clean-up a few loose ends and add some new insights. Let's start by listing them:
+
+1. In the Scala 2 version of the code, a message adapter is used, but the `WrappedPongResponse` message that wraps the response is part of the `Pinger` actor's protocol.
+2. The Dotty solution suffers from the same problem. Here, the `Pinger` actor's protocol is extended with the `PingPong.Pong` message.
```

Comment:
Some comment as above with this point.

Reply:
Idem

Reply:
@ckipp01 Reformulated the two bullet items. Should be better compared to what it was before.

Reply:
resolving the conversation

Reply:
Sounds good! 👍